src/Ocelot/Configuration/Creator/CacheOptionsCreator.cs

```diff
@@ -58,6 +58,7 @@ protected virtual CacheOptions Merge(FileCacheOptions options, FileCacheOptions
         var header = options.Header.IfEmpty(globalOptions.Header).IfEmpty(CacheOptions.Oc_Cache_Control);
         var ttlSeconds = options.TtlSeconds ?? globalOptions.TtlSeconds;
         var enableHashing = options.EnableContentHashing ?? globalOptions.EnableContentHashing;
-        return new CacheOptions(ttlSeconds, region, header, enableHashing);
+        var statusCodes = options.StatusCodeFilter ?? globalOptions.StatusCodeFilter;
```

Comment:
There's a lack of defaulting! Refer to my Idea 2 in the [previous comment](https://github.com/ThreeMammals/Ocelot/pull/2337#issuecomment-3566745978). We could implement defaulting closer to the Dev Complete stage, and I'll help with that.
Finally, I expect the following improvement:
```suggestion
        var statusCodes = options.StatusCodes ?? globalOptions.StatusCodes ?? Ocelot.RecommendedStatusCodes;
```